packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -82,12 +82,16 @@ import {
 import { GA4 } from "./ga";
 import { WALLET_ADAPTER_CORE_VERSION } from "./version";
 import { aptosStandardSupportedWalletList } from "./AIP62StandardWallets/registry";
+import type { AptosConnectWalletConfig } from "@aptos-connect/wallet-adapter-plugin";
 
 export type IAptosWallet = AptosStandardWallet & Wallet;
 
 export interface DappConfig {
   network: Network;
+  /** @deprecated */
```

Comment:
if this is deprecated, what should we put here? https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/apps/nextjs-example/src/components/WalletProvider.tsx#L36

Reply:
ohh I see, 
```
export interface AptosConnectWalletConfig extends Omit<ACDappClientConfig, 'defaultNetworkName'> {
  network?: Network;
  preferredWalletName?: string;
}
```

and `ACDappClientConfig` has a `dappId` prop